### ISSUE TITLE
Zoom reset `CMD+0` does not reset font size to the one in the settings.

### ISSUE THREAD

**Issue by alexeyatbluescape** [warpdotdev/Warp#1580]
### Describe the bug

After zooming in (font zoom) and out for various reasons, I would like to get to my default zoom font configured in the settings (Settings > Appearance > Text - Font Size) by pressing `CMD+0` (in OSX). But the font just goes to `13` ignoring/overriding previously defined values in settings.

### To Reproduce

Steps to reproduce:
1. Go to 'Settings'
2. Click on 'Appearance'
3. Scroll down to 'Text - Font Size'
4. Set it to 20.
5. Exit settings
6. Use `CMD` +`+/-` to play with size
7. Reset zoom `CMT`+`0`

Font goes to 13

### Expected behaviour

Font should go to 20

### Screenshots

https://user-images.githubusercontent.com/79166410/177160017-ff1a8277-b9b6-4b67-b8ec-3a05964b6def.mp4

### Operating System

MacOS

### OS Version

12.4

### Shell Version

zsh 5.8.1 (x86_64-apple-darwin21.0)

### Warp Version

v.0.2022.06.27.09.14.stable_01

### Does this block you from using Warp daily?

No

**Comment by alexeyatbluescape** [comment]
No, just a US layout. I do use a custom keyboard, but I have unplugged it and used Mac one. Same issue in latest verison.

**Comment by cruzdanilo** [comment]
related to #2361
